chart/values.yaml

```
@@ -0,0 +1,148 @@
+affinity:
+  podAntiAffinity:
+    preferredDuringSchedulingIgnoredDuringExecution:
+      - weight: 100
+        podAffinityTerm:
+          topologyKey: kubernetes.io/hostname
+          labelSelector:
+            matchLabels:
+              app.kubernetes.io/component: hedera-explorer
+
+annotations: {}
+
+autoscaling:
+  behavior: {}
+  enabled: false
+  maxReplicas: 3
+  metrics:
+    - type: Resource
+      resource:
+        name: cpu
+        target:
+          type: Utilization
+          averageUtilization: 80
+  minReplicas: 1
+
+# Environment variables to add to the container. The value can either be a string or an EnvVarSource. Evaluated as a template.
+env:
+  VUE_APP_LOCAL_MIRROR_NODE_URL: "https://mainnet-public.mirrornode.hedera.com"
+  # BAR:
+  #   valueFrom:
+  #     secretKeyRef:
+  #       name: '{{ .Release.Name }}-custom'
+  #       key: BAZ
+
+# Add config maps or secrets as environment variables. Evaluated as a template.
+envFrom: []
+# - secretRef:
+#     name: '{{ .Release.Name }}-env'
+
+fullnameOverride: ""
+
+global:
+  namespaceOverride: ""
+
+image:
+  pullPolicy: IfNotPresent
+  pullSecrets: []
+  registry: gcr.io
+  repository: hedera-registry/hedera-mirror-node-explorer
+  tag: latest  # Defaults to the chart's app version
```

It's bad practice to use a static tag. Should leave blank so it uses the chart appVersion, which itself should be kept up to date.

```suggestion
  tag: ""  # Defaults to the chart's app version
```